contributing/workflow/bug_triage_guidelines.rst

```diff
@@ -203,23 +231,46 @@ use the following `labels <https://github.com/godotengine/godot-docs/labels>`__:
 -  *Redirect*: Issues and PRs involving moving content and adding a redirect rule on the backend.
 -  *Website*: Issues related to adding website features and fixing bugs, whether on the front or back-end,
 
-**Topic:**
+Topic:
+^^^^^^
 
 The available topics describe the same content as the topics in the main
 repository.
 
 Milestones
 ~~~~~~~~~~
 
-`Milestones <https://github.com/godotengine/godot/milestones>`_ correspond to
-planned future versions of Godot for which there is an existing roadmap. Issues
-that fit in the said roadmap should be filed under the corresponding milestone;
-if they don't correspond to any current roadmap, they should be left without
-milestone. As a rule of thumb, an issue corresponds to a given milestone if it
-concerns a feature that is new in the milestone, or a critical bug that can't be
-accepted in any future stable release, or anything that Juan wants to work on
-right now. :)
-
-Contributors are free to pick issues regardless of their assigned milestone;
-if a fix is proposed for a bug that was not deemed urgent and thus without
-milestone, it would likely still be very welcome.
+`Milestones <https://github.com/godotengine/godot/milestones>`_ are used for
+some issues and all PRs.
+
+We have milestones for specific minor engine versions, like ``4.5`` and ``4.6``,
+as well as general milestones for major engine versions, like ``3.x`` and
+``4.x``. In the ``godot-proposals`` repo, we also have a ``5.0`` milestone for
+compatibility-breaking changes that will be considered for Godot 5.0, in many
+years.
+
+Issues are assigned to a milestone, such as ``4.5``, if they are related to
+features introduced in that engine version, or are bugs (regressions) in that
+version. Additionally, all issues completed during that engine version are added
+to the milestone. We don't always use the ``4.x`` milestone for issues, since by
+default all issues are related to Godot 4.x. However, we do use the ``3.x``
+milestone to mark issues that are specific to Godot 3.x.
+
+All pull requests are assigned to a milestone. By default, enhancement and
+feature PRs are assigned to the ``4.x`` milestone, and bugs are assigned to the
+current minor version milestone, such as ``4.5``. Towards the end of a minor
+version's development, PRs currently in the minor version milestone are
+reassigned to either the major version milestone (``4.x``), or the next minor
+version milestone (such as ``4.6``).
```

Comment:
This section should make it clear that PRs are reassigned only if they are not considered for the version, as this makes it sound like all PRs are moved before the version is released